src/Cake.AzureDevOps/Pipelines/AzureDevOpsBuild.cs

```diff
@@ -30,6 +30,27 @@ public AzureDevOpsBuild(ICakeLog log, AzureDevOpsBuildSettings settings)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AzureDevOpsBuild"/> class.
+        /// </summary>
+        /// <param name="log">The Cake log context.</param>
+        /// <param name="settings">Settings for accessing AzureDevOps.</param>
+        /// <param name="build">The build.</param>
+#pragma warning disable CS3001 // Argument type is not CLS-compliant
+        public AzureDevOpsBuild(ICakeLog log, AzureDevOpsBuildsSettings settings, Build build)
```

Comment:
This should be internal not public, since it otherwise exposes the AzureDevOps API. As soon as it is internal disabling the warning no longer should be required.